src/main/java/ee/carlrobert/codegpt/completions/HuggingFaceModel.java

```diff
@@ -4,47 +4,46 @@
 
 import java.net.MalformedURLException;
 import java.net.URL;
-import java.util.regex.Pattern;
 
 public enum HuggingFaceModel {
 
-  CODE_LLAMA_7B_Q3(7, 3, "codellama-7b-instruct.Q3_K_M.gguf"),
```

Comment:
I guess you made a mistake while rebasing/merging, these changes should not be reverted, they were part of the https://github.com/carlrobertoh/CodeGPT/pull/529 fix

Reply:
As I commented, the links were broken, now every URL is working again.

Reply:
But if you insist, I reapply master, not my responsibility then ;)

Reply:
The `modelName` is used to store the selected model in the XML config, so we can't have different models having the same `modelName`, e.g:
https://github.com/carlrobertoh/CodeGPT/blob/eb595a73c0a4ca81753d5248a622be8be16537c0/src/main/java/ee/carlrobert/codegpt/completions/HuggingFaceModel.java#L10-L11

We either have to manually concatenate the quantization to the `toString` value or introduce a new field for the download directory.

Reply:
Done, so all URLs are working except TheBloke (new master), good night 😅

Reply:
´toString` could probably also just return `name()`, since the enum names are unique

Reply:
Why do you merge such a breaking change without testing it, at least manually?
```java
public static void main(String[] args) {
  java.util.Arrays.stream(values())
    .collect(java.util.stream.Collectors.groupingBy(e -> e.directory, java.util.stream.Collectors.toList()))
    .values().stream().map(e -> e.get(0)).sorted()
    .forEach(e -> System.out.println(e.getFileURL()));
}

```

Reply:
As I said I'm sorry I didn't notice the breaking change, @carlrobertoh also didn't notice it when reviewing it, it also hasnt been released yet either, so no need to make a big fuss out of it 😄